Purchases/Purchasing/StoreKitAbstractions/SK1StoreTransaction.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SK1StoreTransaction.swift
+//
+//  Created by Nacho Soto on 1/4/22.
+
+import StoreKit
+
+internal struct SK1StoreTransaction: StoreTransactionType {
+
+    init(sk1Transaction: SK1Transaction) {
+        self.underlyingSK1Transaction = sk1Transaction
+
+        self.productIdentifier = sk1Transaction.productIdentifier ?? ""
+        self.purchaseDate = sk1Transaction.purchaseDate
+        self.transactionIdentifier = sk1Transaction.transactionID
+    }
+
+    let underlyingSK1Transaction: SK1Transaction
+
+    let productIdentifier: String
```

Comment:
I think we could add `quantity` - in SK1 it comes from the payment and SK2 it's just from the transaction. 
We currently only allow setting to 1, but we intend to add multi-quantity support

Reply:
that said, it can be done later

Reply:
👍 doing that separately